src/com/massivecraft/factions/cmd/CmdFactionsPowerBoost.java

```diff
@@ -33,6 +36,7 @@ public CmdFactionsPowerBoost()
 		// Parameters
 		this.addParameter(TypeString.get(), "p|f|player|faction");
 		this.addParameter(parameterMplayer);
+		this.addParameter(TypeString.get(), "a|r|s|add|remove|set");
```

Comment:
To clear this up, let me give you an example:
Let's say your Type is called Type givetype,  then this line would read 

'this.addParameter(TypeGiveType.get(), "add|remove|set");

And line 65 would simply be GiveType givetype = this.readArgAt(2);